### PR TITLE
Cornerstone Classic custom headlines are not being marked for SEO significance

### PR DESCRIPTION
**Pull request by kagg-design**
https://onthegosystems.myjetbrains.com/youtrack/issue/wpmltm-3307

### CODE REVIEW

**Comment by kagg-design** [comment]
Please review the first commit only.

**Review by strategio — DISMISSED** [review]
+1 @kagg-design

File: src/class-wpml-cornerstone-translatable-nodes.php [hunk 1]
```diff
@@ -143,6 +143,10 @@ private function get_wrap_tag( $settings ) {
 			return $settings['text_tag'];
 		}
 
+		if ( isset( $settings['_type'] ) && 'classic:custom-headline' === $settings['_type'] && isset( $settings['level'] ) ) {
+			return $settings['level'];
+		}
+
```
Comment (brucepearson) [inline src/class-wpml-cornerstone-translatable-nodes.php:152]: @kagg-design 

How about reducing the duplication:
```
$seo_elements = array( 
    'headline' => 'text_tag',
    'classic:custom-headline' => 'level'
);

foreach( $seo_elements as $type => $attr ) {
    if ( isset( $settings['_type'] ) &&$type === $settings['_type'] && isset( $settings[ $attr ] ) ) {
			return $settings[ $attr ];
		}
}
```

**Review by brucepearson — CHANGES_REQUESTED** [review]

File: src/class-wpml-cornerstone-translatable-nodes.php (hunk 1 above)
Comment (kagg-design) [reply src/class-wpml-cornerstone-translatable-nodes.php:152]: @brucepearson Good point, thanks, done.

**Review by brucepearson — DISMISSED** [review]
@kagg-design Looks good. +1

**Review by strategio — DISMISSED** [review]
+1 @kagg-design

**Pull request closed without merging**